<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatapickerImageWidget</class>
 <widget class="QWidget" name="DatapickerImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="1" colspan="2">
        <widget class="QLabel" name="lPlot">
         <property name="text">
          <string>Image:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="6">
        <widget class="QSpinBox" name="sbPointSeparation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="6">
        <widget class="NumberSpinBox" name="sbTernaryScale">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="1" colspan="2">
        <widget class="QLabel" name="lTernaryScale">
         <property name="text">
          <string>x+y+z=</string>
         </property>
        </widget>
       </item>
       <item row="17" column="5">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="2">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="1" colspan="2">
        <widget class="QLabel" name="lMinSegmentLength">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1" colspan="3">
        <widget class="QLabel" name="lSegments">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Segments</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLineEdit" name="leName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="leFileName">
           <property name="toolTip">
            <string>Specify the name of the image file.</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bOpen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string> Select the image file</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QComboBox" name="cbGraphType">
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="6">
        <widget class="NumberSpinBox" name="sbRotation">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="16" column="6">
        <widget class="QSpinBox" name="sbMinSegmentLength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="17" column="1" colspan="4">
        <widget class="QLabel" name="lPointSeparation">
         <property name="text">
          <string>Point separation:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="18" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>231</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QLabel" name="lRotation">
         <property name="text">
          <string>Rotation:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="6" rowspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="5">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="4">
          <widget class="NumberSpinBox" name="sbPositionY3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lPositionY2">
           <property name="text">
            <string>y=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="NumberSpinBox" name="sbPositionY1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="NumberSpinBox" name="sbPositionZ1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="NumberSpinBox" name="sbPositionY2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="NumberSpinBox" name="sbPositionZ2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lPositionY1">
           <property name="text">
            <string>y=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="NumberSpinBox" name="sbPositionX1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="lPositionZ3">
           <property name="text">
            <string>z=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lPositionX2">
           <property name="text">
            <string>x=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="NumberSpinBox" name="sbPositionX3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lPositionX3">
           <property name="text">
            <string>x=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="lPositionY3">
           <property name="text">
            <string>y=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="NumberSpinBox" name="sbPositionZ3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="lPositionZ2">
           <property name="text">
            <string>z=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="NumberSpinBox" name="sbPositionX2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Spinbox for numbers.</string>
           </property>
           <property name="feedback" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lPositionX1">
           <property name="text">
            <string>x=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="lPositionZ1">
           <property name="text">
            <string>z=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLabel" name="lPlotType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1" colspan="4">
        <widget class="QLabel" name="lRefPoint">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reference Points</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="lRefPoint1">
         <property name="text">
          <string>1.</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="lRefPoint2">
         <property name="text">
          <string>2.</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QRadioButton" name="rbRefPoint1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QRadioButton" name="rbRefPoint2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="lRefPoint3">
         <property name="text">
          <string>3.</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QRadioButton" name="rbRefPoint3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tSymbol">
      <attribute name="title">
       <string>Ref. Points Symbol</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QCheckBox" name="chbSymbolVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tEdit">
      <attribute name="title">
       <string>Edit Image</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="11" column="2">
        <widget class="QWidget" name="wForegroud" native="true"/>
       </item>
       <item row="2" column="2">
        <widget class="QWidget" name="whHue" native="true"/>
       </item>
       <item row="5" column="2">
        <widget class="QWidget" name="wSaturation" native="true"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lIntensity">
         <property name="text">
          <string>Intensity:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QWidget" name="whValue" native="true"/>
       </item>
       <item row="10" column="2">
        <widget class="QWidget" name="whForeground" native="true"/>
       </item>
       <item row="8" column="2">
        <widget class="QWidget" name="whIntensity" native="true"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cbPlotImageType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QWidget" name="wIntensity" native="true"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lValue">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lHue">
         <property name="text">
          <string>Hue:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lPlotImageType">
         <property name="text">
          <string>Current image:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QWidget" name="whSaturation" native="true"/>
       </item>
       <item row="3" column="2">
        <widget class="QWidget" name="wHue" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lSaturation">
         <property name="text">
          <string>Saturation:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lForeground">
         <property name="text">
          <string>Foreground:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QWidget" name="wValue" native="true"/>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>cbGraphType</tabstop>
  <tabstop>sbMinSegmentLength</tabstop>
  <tabstop>sbPointSeparation</tabstop>
  <tabstop>chbSymbolVisible</tabstop>
  <tabstop>cbPlotImageType</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
